models/book.js

```diff
@@ -0,0 +1,57 @@
+//vars used for global scope availability
+var app = app || {};
+// noinspection JSVoidFunctionReturnValueUsed
+app.BookModel = Backbone.Model.extend(
+	{
+		initialize : function (options) {
+		},
+		/**
+		 * Sets url path for API requests
+		 */
+		url: function () {
+			let myURL = document.location.origin + '/wp-json/bookAPI/v1/book';
+			if (this.id) {
+				myURL += '/' + this.id;
+			}
+			return myURL;
+		},
+		/**
+		 * Default values<br>
+		 * post_id is by default the id of the current book instance
+		 */
+		defaults: {
+			post_id : jQuery( '#get_page_id' ).val(),
+			title: '',
+			author: '',
+			genre: '',
+			summary: '',
+		},
+		/**
+		 * Checks if author has 2 words
+		 * @param attrs
+		 * @param options
+		 * @returns {string}
+		 */
+		validate : function (attrs, options) {
+			if (this.get( 'author' ).split( ' ' ).length < 2) {
+				return "author must be a valid name";
```

Comment:
This string  and any others have to be translatable. Write me on slack regarding this.
```suggestion
				return YOUR_BLOBAL_LOCALISED_VAR.t.invalid_autor;
```

Reply:
I localized it for now